### DISCUSSION TITLE
Remove @import from css file?

### DISCUSSION THREAD

**Discussion by jf-home** [FullHuman/purgecss#980] [Q&A]
Hi,

I want to strip out an import statement in a CSS file at build time, can anyone offer advice as to how to go about that (or whether this could be incorporated as a future feature of purgecss)?

The full import statement is as follows:

`@import url(https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap);`

The reason for wanting to remove the import is because it's not needed (in my case) and, as it's an externally-referenced file, it's hitting the page performance due to blocking page rendering.

Most grateful for any help or advice!

**Comment by meyshad** [comment]
If you need to delete or comment on the file, you can access it later.
But if you need a file and don't want to use import, you can put the external file in the internal files.
